src/components/VHeader/VHeader.vue

```diff
@@ -1,37 +1,44 @@
 <template>
   <header
-    class="sticky top-0 flex py-4 px-4 md:px-7 items-center justify-between z-40 w-full bg-white gap-x-2"
+    class="fixed top-0 flex py-4 px-4 md:px-7 items-stretch z-40 w-full bg-white gap-x-2"
```

Comment:
If I switch this back to `sticky` and remove the hardcoded margins from the default layout, everything appears to work fine. That seems preferable. Do you remember the reason to switch it to fixed?

Reply:
When one of the parents has an overflow value set to anything but `visible`, the `sticky` stops working: https://css-tricks.com/dealing-with-overflow-and-position-sticky/ 
And we set overflow to hidden on `app` div in the default layout

Reply:
I realized with some modification it _actually_ doesn't need sticky _or_ fixed, because our grid layout is keeping at the top! See #609

Reply:
I wish I read this comment before trying to figure out why it didn't have sticky 😆